server.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express');
+const bcrypt = require('bcrypt');
+const bodyParser = require('body-parser');
+
+const app = express();
+
+app.use(bodyParser.json());
+
+app.post('/api/login', async (req, res) => {
+  const { email, password } = req.body;
+
+  // This is a placeholder for fetching user data from a database
+  const user = getUserFromDatabase(email);
+
+  if (user) {
+    const match = await bcrypt.compare(password, user.password);
+
+    if (match) {
+      res.status(200).json({ message: 'Login successful' });
+    } else {
+      res.status(401).json({ message: 'Invalid credentials' });
+    }
+  } else {
```

Comment:
Consider using a middleware for handling errors and sending responses. This can make your code more modular and easier to maintain.